mlflow/ml-package-versions.yml

```diff
@@ -205,7 +205,8 @@ catboost:
     install_dev: |
       # TODO: Remove the commit once https://github.com/catboost/catboost/issues/2242 is fixed
       pip install \
-        git+https://github.com/catboost/catboost.git@f8921d839eccf11e533a46347dde5b5b25b17190#subdirectory=catboost/python-package
+        git+https://github.com/catboost/catboost.git@f8921d839eccf11e533a46347dde5b5b25b17190#subdirectory=catboost/python-package \
```

Comment:
Is https://github.com/catboost/catboost/issues/2242 fixed?

Reply:
We can remove `install_dev` if it's challenging to build catboost from the soruce.

Reply:
Seems it's still broken :)

Reply:
We don't have all of the build tools installed required to compile the cython libraries. There's a comment in Issue 2242 about the updated manual install process. Can we add those steps for the linux environment to ensure we have the required DLLs for the CLI install command to use?